src/api/transfers/routes.js

```diff
@@ -37,7 +37,14 @@ const Enum = require('@mojaloop/central-services-shared').Enum
 const validateIncomingErrorCode = require('@mojaloop/central-services-error-handling').Handler.validateIncomingErrorCode
 
 const tags = ['api', 'transfers', Enum.Tags.RouteTags.SAMPLED]
-const transferState = [Enum.Transfers.TransferState.RECEIVED, Enum.Transfers.TransferState.RESERVED, Enum.Transfers.TransferState.COMMITTED, Enum.Transfers.TransferState.ABORTED, Enum.Transfers.TransferState.SETTLED]
+const transferState = [
+  Enum.Transfers.TransferState.RECEIVED,
+  Enum.Transfers.TransferState.RESERVED,
+  Enum.Transfers.TransferState.COMMITTED,
+  Enum.Transfers.TransferState.ABORTED,
+  Enum.Transfers.TransferState.SETTLED
+  // todo: remove SETTLED value
```

Comment:
Can you add some more information to this comment? About your observations...

Reply:
Added